packages/browser/src/index.ts

```diff
@@ -16,6 +17,7 @@ const INTEGRATIONS = {
   ...windowIntegrations,
   ...CoreIntegrations,
   ...BrowserIntegrations,
+  ...PluggableIntegrations,
```

Comment:
h: This will not tree-shake as soon as people import `Integrations` (and not even access one of the pluggable ones) because the spread operator (or in fact assigning an object) is a runtime operation. It's for the same reason we have `BrowserTracing` as a direct export and not under `Integrations`.

We need to do the same here.

Reply:
right, that makes sense of course... I wonder actually if we should then completely get rid of the `Integrations` key and move _all of these_ to the top level instead? 🤔 And then maybe suffix them with `Integration` or something like this...:

```
export { Dedupe as DedupeIntegration } from '@sentry/integrations';
// etc
```

IMHO it would be a bit weird to have the default integrations be available under `Integrations` and the pluggable ones directly at the root 😅

Reply:
In the next major we should remove the Integrations object for sure. I agree having them in separate places is weird. I think we could export everything top level.

Reply:
https://github.com/getsentry/sentry-javascript/issues/8844